testing/kokoro/system_tests.sh

```diff
@@ -98,7 +98,13 @@ export GOLANG_SAMPLES_FIRESTORE_PROJECT=golang-samples-fire-0
 
 set -x
 
-source ${KOKORO_GFILE_DIR}/secret_manager/go-aws-secrets
+export STS_AWS_SECRET=`gcloud secrets versions access latest --project cloud-devrel-kokoro-resources --secret=go-storagetransfer-aws`
```

Comment:
Why is this updated for AWS keys here as well?

Reply:
We actually updated the AWS creds recently, at the same time we created Azure creds.

Basically, before we used AWS creds which were owned by a particular Googler and were just attached to his gcard. Now, they're owned by the data transfer service team, and we're retiring those old creds